src/components/IllustratedHeaderPageLayout.tsx

```diff
@@ -16,9 +16,12 @@ type IllustratedHeaderPageLayoutProps = HeaderPageLayoutProps & {
 
     /** Overlay content to display on top of animation */
     overlayContent?: () => ReactNode;
+
+    /** TestID to apply to the whole section container */
+    testID?: string;
```

Comment:
Let's make this a required param?

```suggestion
    testID: string;
```